packages/syft/src/syft/service/worker/worker_image.py

```diff
@@ -91,6 +101,10 @@ class SyftWorkerImage(SyftObject):
     image_hash: Optional[str]
     created_at: DateTime = DateTime.now()
     created_by: SyftVerifyKey
+    source_file: Optional[str]
```

Comment:
Q: Is source file attribute being used, I could find it being used?

Reply:
It's being used when user pass in a Dockerfile and now it holds the name of that file

Reply:
Hmm I still don't see where this attribute being used?

Reply:
It's only used if we pass the a `.dockerfile`. For now in the api notebook we pass directly the string so we are not using it yet

Reply:
If we're not using it yet we should not include it in this PR. We should only add it in when we use it and implement support for it.

Reply:
This shouldn't be tracked at SyftWorkerImage level, this is more of a WorkerConfig thing.